docs/source/cpp/compute.rst

```diff
@@ -286,6 +286,8 @@ an ``Invalid`` :class:`Status` when overflow is detected.
 +--------------------------+------------+--------------------+---------------------+
 | power_checked            | Binary     | Numeric            | Numeric             |
 +--------------------------+------------+--------------------+---------------------+
+| sign                     | Unary      | Numeric            | Int8                |
```

Comment:
Should add a note explaining how the output is computed. You can look at other sections for examples.

Reply:
I added a "Notes" column to the table, and included notes describing the output values and special case of 0.